Sources/MapboxMaps/Style/Types/ExpressionOptions.swift

```diff
@@ -64,14 +64,21 @@ public struct FormatOptions: Codable, Equatable, ExpressionArgumentConvertible {
         return [.option(.format(self))]
     }
 
-    public init(fontScale: Double? = nil, textFont: [String]? = nil, textColor: UIColor? = nil) {
+    public init(fontScale: Double? = nil, textFont: [String]? = nil, textColor: Value<StyleColor>? = nil) {
         self.fontScale = fontScale
         self.textFont = textFont
 
         if let textColor = textColor {
-            self.textColor = StyleColor(textColor)
+            self.textColor = textColor
         }
```

Comment:
why not just assign textColor here? we don't need textColor to be non-nil, this check belongs to the other initializer where the parameter for `textColor` is a `UIColor`